pkg/addons/config.go

```diff
@@ -161,4 +164,9 @@ var Addons = []*Addon{
 		set:       SetBool,
 		callbacks: []setFn{enableOrDisableAddon},
 	},
+	{
+		name:      "metadata",
```

Comment:
This should be called 'gcp-metadata', to disambiguate it from other sources.

Reply:
I'm going to change it to gcp-auth for the time being.